L1Trigger/TrackFindingTracklet/python/Demonstrator_cfi.py

```diff
@@ -3,9 +3,9 @@
 
 TrackTriggerDemonstrator_params = cms.PSet (
 
-  LabelIn  = cms.string( "TrackFindingTrackletProducerTBout"  ), #
-  LabelOut = cms.string( "TrackFindingTrackletProducerKFin" ), #
-  DirIPBB  = cms.string( "/heplnw039/tschuh/work/proj/kfin/" ), # path to ipbb proj area
+  LabelIn  = cms.string( "TrackFindingTrackletProducerIRin"  ), #
+  LabelOut = cms.string( "TrackFindingTrackletProducerTBout" ), #
+  DirIPBB  = cms.string( "/heplnw039/tschuh/work/proj/IRinTBout/" ), # path to ipbb proj area
```

Comment:
Could this be changed to a generic path as anyone else using this code has to change it to their own working directory:
This could be done by adding: 
`import os`
`CMSSW_BASE = os.getenv('CMSSW_BASE')`

To the start of the file and then changing to 
`DirIPBB = cms.string(CMSSW_BASE+"/src/L1Trigger/)` 

I don't know how many people will be running this in an ipbb workflow style? Or if anyone apart from me and you are running this.....